docs/publishing.md

```diff
@@ -87,3 +110,14 @@ Aside from publishing your content immediately, you can make sure your content i
     Activate it to remove the previously selected publication date.
 
     ![Discard Publish Date button in the menu](img/discard_publish_date.png "Discard Publish Date button in the menu")
+
+    ### Timeline
+
+    The timeline in Page mode shows all changes that will happen to the Page in the future.
+
+    [screenshot]
+
+    You can use the slider to preview what the page will look like at a given time.
```

Comment:
the page -> your website? As Page is an official name in eZ.

at any given time